rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/DBFileMap.kt

```diff
@@ -160,8 +160,11 @@ fun Expression.createKList(fileMetadata: FileMetadata, interpreter: InterpreterC
     return if (type() is KListType) {
         interpreter.toSearchValues(this, fileMetadata)
     } else {
-        val value = interpreter.eval(this)
-        listOf(value.asString(fileMetadata.accessFieldsType[0]))
+        when (val value = interpreter.eval(this)) {
+            is StartValValue -> listOf(fileMetadata.fields.first().fieldName)
```

Comment:
I don't think this is the correct implementation. If ChatGPT isn't misleading me, this code essentially means:
```
     C     *START        SETLL     TABDS01L
     C     *END          SETLL     TABDS01L
```
This moves the cursor to the beginning and then to the end of the file. However, this feature should actually be supported by Reload (an open-source project that implements record-level access APIs).

I believe the correct implementation would throw a runtime error with the message:
`*START|*END constant not yet supported.`

Reply:
Reading the documentation again after this comment I think you are right. I apologize for the mistake, the documentation is very messy.

However we can fix easily by reverting the check on `StartValValue` and `EndValValue` and forwarding directly `*START` and `*END` to reload. At that point it should already return the runtime error you mentioned until Reload implements this feature.

Reply:
It appears that enacting the changes I suggested in my previous comment no runtime error is thrown. This appears to be the case with every string we forward to the `setll` method. Am I missing something? Was this behaviour expected? I pushed the new code to allow for further analysis, comparison or review.

Reply:
You aren't missing anything.

This is the interface that Jariko uses, as defined in Reload:  
https://github.com/smeup/reload/blob/master/base/src/main/kotlin/com/smeup/dbnative/file/DBFile.kt

I believe Reload does not take care the value passed to the `setll` function, but that's not entirely wrong because `*START` and `*END` are not strings; they differ from `"*START"` and `"*END"`.

Therefore, until we decide how Reload should handle this feature, it's correct for Jariko to throw an error.

Reply:
That is legitimate. Considering this, I am going to update the code as follow:
- Throw an error at runtime when `*START` or `*END` are with `SETLL`
- Mark the current test as ignored
- Create a clone/similar error test that expects the runtime error
- Put a note on the error test regarding why it throws an error (not implemented yet by Reload) and what to do when it gets solved

Reply:
I like it